account/list_pnids_rpc.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package account;
+
+option go_package = "./;grpc_account";
+
+// TODO - Anything missing? Is this a good order for the data?
+message PNID {
+	bool deleted = 1;
+	string environment = 2;
+	uint32 pid = 3;
+	string username = 4;
+	string role = 5;
+	Mii mii = 6;
+	string birthday = 7;
+	string gender = 8;
+	string country = 9;
+	string language = 10;
+	uint32 region = 11; // * See https://gist.github.com/jonbarrow/15eb9665c474d6d0a2679da4691d2886 for region ID details
+	string timezone_name = 12;
+	uint32 timezone_offset = 13;
+	bool active_flag = 14; // TODO - What are these flags? Rename these?
+	bool marketing_flag = 15; // TODO - What are these flags? Rename these?
+	bool off_device_flag = 16; // TODO - What are these flags? Rename these?
```

Comment:
`marketing` was originally used for sending newsletters to the user's email.
`off_device` was originally used for allowing or disallowing signing into the account from other devices that aren't the linked console (e.g. a computer).

can't remember about `active_flag` though

Reply:
I tried to look up `active_flag` on Discord and only found 2 messages it's ever mentioned in, one from 2019 and one from 2018, and neither have any information